pkg/manager/manager.go

```diff
@@ -119,6 +119,10 @@ func (m *SupportBundleManager) Run() error {
 			types.ManagerPhaseClusterBundle,
 			m.phaseCollectClusterBundle,
 		},
+		{
+			types.ManagerPhasePrometheusBundle,
+			m.phaseCollectPrometheusBundle,
```

Comment:
@Yu-Jack Can we continue to the next phase even if there is an error in this phase? Node bundles are very important and we can ignore the failure in Prometheus collection.

Reply:
Sure, NP. I'll handle this.

Reply:
@bk201 I made this optional, so I slightly changed the structure where runs the phase.